src/commcare_cloud/ansible/roles/pgbouncer/templates/pgbouncer.ini.j2

```diff
@@ -36,7 +36,10 @@ listen_port = {{ pgbouncer_port }}
 ;unix_socket_dir = /tmp
 ;unix_socket_mode = 0777
 ;unix_socket_group =
-unix_socket_dir = /var/run/postgresql
+unix_socket_dir = {{ pgbouncer_socket_dir }}
```

Comment:
In my testing, I had to create this directory, does this playbook work if that directory does not already exist? For example if we set this to 3 processes?

Reply:
It does not at the moment, but we're also not using unix sockets at all (right?).
We can decide if we want to expand this part and deal with the directories, or just deactivate it explicitly and drop it.

Reply:
Turns out we may need them for some features, I've added the directory creation in the last commit.